text/chapter11.md

```diff
@@ -85,19 +85,21 @@ The game is very simple, but the aim of the chapter is to use the `purescript-tr
 
 We will start by looking at some of the monads provided by the `purescript-transformers` package.
 
-The first example is the `State` monad, which provides a way to model _mutable state_ in pure code. We have already seen two approaches to mutable state provided by the `Eff` monad, namely the `REF` and `ST` effects. `State` provides a third alternative, but it is not implemented using the `Eff` monad.
+The first example is the `State` monad, which provides a way to model _mutable state_ in pure code. We have already seen an approach to mutable state provided by the `Effect` monad. `State` provides an alternative.
 
 The `State` type constructor takes two type parameters: the type `s` of the state, and the return type `a`. Even though we speak of the "`State` monad", the instance of the `Monad` type class is actually provided for the `State s` type constructor, for any type `s`.
 
 The `Control.Monad.State` module provides the following API:
 
 ```haskell
-get    :: forall s.             State s s
-put    :: forall s. s        -> State s Unit
-modify :: forall s. (s -> s) -> State s Unit
+get     :: forall s.             State s s
+gets    :: forall s. (s -> a) -> State s a
+put     :: forall s. s        -> State s Unit
+modify  :: forall s. (s -> s) -> State s s
+modify_ :: forall s. (s -> s) -> State s Unit
 ```
 
-This looks very similar to the API provided by the `REF` and `ST` effects. However, notice that we do not pass a mutable reference cell such as a `Ref` or `STRef` to the actions. The difference between `State` and the solutions provided by the `Eff` monad is that the `State` monad only supports a single piece of state which is implicit - the state is implemented as a function argument hidden by the `State` monad's data constructor, so there is no explicit reference to pass around.
+Note that these API signatures are presented in a simplified form using the `State` type constructor for now. The actual API involves `MonadState` which we'll cover in the later "Type Classes" section of this chapter, so don't worry if you see different signatures in your IDE tooltips or on Pursuit.
```

Comment:
I think this note is important so readers aren't confused by IDE tooltips. I wasted a bunch of time assuming this was another part of the documentation that went out of date.